content/en/docs/gke/deploy/management-setup.md

```diff
@@ -138,7 +169,7 @@ For a more detailed explanation of the drastic changes happened in vKubeflow 1.1
 
```

Comment:
for a11y: line 168

-> "and check the `./build/cnrm-install-*` folders." or something like that